packages/eslint-plugin/src/rules/no-unnecessary-type-parameters.ts

```diff
@@ -21,7 +21,7 @@ export default createRule({
       recommended: 'strict',
     },
     messages: {
-      sole: 'Type parameter {{name}} is used only once.',
+      sole: 'Type parameter {{name}} is used only once in the function signature.',
```

Comment:
@kirkwaiblinger I added _"function"_ because I think some users won't know what _"signature"_ means.

Reply:
> @kirkwaiblinger I added _"function"_ because I think some users won't know what _"signature"_ means.

Yep, no opinion on this particular minutia 👍 

***

I wonder, though, don't we need to be a little careful to get the node type correct?

The rule reports on types that aren't necessarily in function/method signatures, for example: 
```ts
        class Joiner<T extends string | number> {
          join(el: T, other: string) {
            return [el, other].join(',');
          }
        }
```

Reply:
Ah true... hmm........

Reply:
Switched the descriptor to _"class signature"_ for those.